README.md

```diff
@@ -9,3 +9,30 @@ For more information about the control container, including how to use it and ho
 
 You'll need Docker 17.06.2 or later, for multi-stage build support.
 Then run `make`!
+
+## Connecting to AWS Systems Manager (SSM)
+
+Starting from v0.5.0, users have the option to pass in their own activation information for SSM.
+This is for users that want to set up on-premises virtual machines (VMs) in their hybrid environment as managed instances.
+
+Users can add their [own activations](https://docs.aws.amazon.com/systems-manager/latest/userguide/sysman-managed-instance-activation.html) by populating the control container's user data with a base64-encoded JSON block.
+
+To use hybrid activations for managed instances you will want to generate a JSON-structure like this:
+
+```
+{
+  "ssm": {
+    "activation-id": "foo",
+    "activation-code": "bar",
+    "region":"us-west-2"
+  }
+}
+```
+
+Once you've created your JSON, you'll need to base64-encode it and set it as the value of the control host container's `user-data` setting in your [instance user data toml](https://github.com/bottlerocket-os/bottlerocket#using-user-data).
```

Comment:
```suggestion
Once you've created your JSON, you'll need to base64-encode it and put it in the control container's `user-data` setting in your [instance user data](https://github.com/bottlerocket-os/bottlerocket#using-user-data).
For example:
```

Reply:
That sentence is still complex, so I do think the friendly "For example:" afterward helps :)